sys-tuner/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+use crate::main::SOLANA_SYS_TUNER_PATH;
+use unix_socket::UnixStream;
+
+pub mod main;
+
+pub fn request_system_tuning() {
```

Comment:
wdyt about the name `request_realtime_poh()`

Reply:
Won't this also trigger the rmem/wmem related tweaks? It does make more sense to call it `request_realtime_poh` from `poh_service` perspective.